doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -115,10 +115,11 @@ you will need to rebuild Chapel from source in a different configuration.
       # run the sample program
       ./hello
 
-   At this point, you can choose to continue using ``setchplenv.bash``
-   or use another method to configure Chapel. In particular, you might
-   use a :ref:`chplconfig <readme-chplenv.chplconfig>` file or
-   set environment variables your in dot files or your own script.
+   Note that these environment settings will be lost outside of this terminal session.
+   You can choose to source ``setchplenv.bash`` whenever you want to use relevant commands.
+   If you want to automate this setup, refer to :ref:`chplenv.rst <readme-chplenv.recommended_settings>`
```

Comment:
Doesn't this render "chplenv.rst" on the generated doc page? I think it should say "Setting up your Chapel Environment" or just "Chapel Environment" or something else.

Reply:
Reading the rest of the doc I can see some other examples of this. Still I think we can do better than that while changing this paragraph here

Reply:
Historically we have kept references to other parts of the documentation as filenames to better support users who are reading the documentation offline in a text editor (primary use-case being a user on an HPC system). 

I agree with your suggestions, but think we should take that discussion up in a separate issue/effort.

Reply:
Yeah that makes sense.